docs/modules/ROOT/pages/index.adoc

```diff
@@ -852,179 +850,116 @@ The `validFrom` field MUST exist and it MUST express the date and time when the
-
-[#vc-credentialsubject-affiliations]
-===== Affiliations
-
-The `affiliations` property is an OPTIONAL field within the `credentialSchema` property. If present, it MUST be an array in which each item MUST contain information about an affiliation of the _<<_named_actor,named actor>>._ The process of proving the validity of these affiliations is outside the scope of this specification. This specification mentions at least three properties that MAY be used to represent the _<<_named_actor,named actor’s>>_ affiliation details: `id`, `name`, and `proof`.
-
-.Example:
-[source,json]
-----
-"credentialSubject": {
-  ...
-  "affiliations": [
+      },
+      "verifiedAt": "2024-07-26T22:30:15Z"
+    },
     {
-      "id": "https://example-affiliated-organization.com",
-      "name": "Example Affiliated Organization",
-      "proof": "https://example-affiliated-organization.com/proofs/ck4592p5lk8u05mdg8bg5ac7ishlqfh1"
-    }
-  ]
-  ...
-}
-----
-
-[#vc-credentialsubject-affiliation-id]
-Affiliation ID:::
-The `affiliations[?].id` MUST be present and MUST be a valid URI that, when dereferenced, MUST result in a proof of authenticity of the _<<_named_actor,named actor's>>_ affiliated entity.
-
-[#issue-149]
-NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/149[issue #149]): Define what we mean by "proof of authenticity" here.
-
-[#vc-credentialsubject-affiliation-name]
-Affiliation name:::
-The `affiliations[?].name` MUST be present and MUST NOT be empty. The `affiliations[?].name` property is the name of the  _<<_named_actor,named actor's>>_ affiliated entity.
-
-[#vc-credentialsubject-affiliation-proof]
-Affiliation proof:::
-The `affiliations[?].proof` is an OPTIONAL field. If present, it MUST be a valid URI that, when dereferenced, MUST result in the proof of authenticity of the  _<<_named_actor,named actor's>>_ affiliation validity, i.e., the _<<_named_actor,named actor>> is affiliated indeed with this organization. The content as well as the process of validatig an affiliation relation are outside the scope of this specification.
-
-[#vc-credentialsubject-identities]
-===== Identities
-
-The `identities` property is an optional field within the `credentialSchema` property. If present, it MUST be an array containing information about the accounts that the _<<_named_actor,named actor>>_ requested to be bound with the _<<C2PA asset>>_ being described.
-
-.Example:
-[source,json]
-----
-"credentialSubject": {
-  ...
-  "identities": [
+      "type": "cawg.affiliation",
+      "provider": {
+        "id": "https://example-affiliated-organization.com",
+        "name": "Example Affiliated Organization",
+        "proof": "https://example-affiliated-organization.com/proofs/ck4592p5lk8u05mdg8bg5ac7ishlqfh1"
```

Comment:
is this confusing ? the proof the is not about the veridicity of the provider itself but rather the identity verification

Reply:
Can you suggest a way to make it less confusing?

Reply:
We could pull  the proof at the same level as verifiedAt ?

Reply:
Continuing this discussion in issue #160